code/game/turfs/floor_attackby.dm

```diff
@@ -583,7 +582,7 @@ var/mining_in_progress = null // Define the variable out of any scopes and the a
 				return
 			// Set mining_in_progress to TRUE to indicate the process has started.
 			mining_in_progress = TRUE
-			visible_message("<span class = 'notice'>[user] starts to break the rocky floor with \the [C.name].</span>", "<span class = 'notice'>You start to break the rocky floor with \the [C.name].</span>")
+			visible_message(SPAN_NOTICE("[user] starts to break the rocky floor with \the [C.name]."), SPAN_NOTICE("You start to break the rocky floor with \the [C.name]."))
```

Comment:
This should be set to user.visible_message(

Reply:
result is the same